README.md

```diff
@@ -16,3 +16,86 @@ The project and Python dependencies can be installed from source with:
+| CLI File | Configuration that is loaded from one or more explicitly provided configuration files through the CLI. |
+| CLI      | Configuration that is loaded from the command line. |
+
+The system produces an error if a global or local configuration file is unreadable (but not missing), or if a CLI-specified file is unreadable or missing.
+
+#### Global Configuration
+
+The global configuration file defaults to `<platform-specific user configuration location>/edq-config.json`.
+The configuration location is chosen according to the [XDG standard](https://en.wikipedia.org/wiki/Freedesktop.org#Base_Directory_Specification) (implemented by [platformdirs](https://github.com/tox-dev/platformdirs)). 
+Below are examples of user-specific configuration file paths for different operating systems:
+ - Linux -- `/Users/<user>/.config/edq-config.json`
+ - Mac -- `/Users/<user>/Library/Application\ Support/edq-config.json`
+ - Windows -- `C:\Users\<user>\AppData\Local\edq-config.json`
+
+The default global configuration location can be changed by passing a path to `--config-global` through the command line.
+This type of configuration is best suited for options that follow the user across multiple projects e.g., login credentials.
+
+#### Local Configuration
+
+Local configuration files are searched in multiple locations, the first file found is used.
+The local config search order is:
+1. `edq-config.json` in the current directory.
+2. A legacy file in the current directory (only if a legacy file is preconfigured).
+3. `edq-config.json` in any ancestor directory on the path to root (including root itself).
+
+#### CLI-Specified Config Files
+
+Any files passed via `--config-file` will be loaded in the order they appear on the command line.
+Later files will override options from previous ones.
+
+Below is an example of a CLI specified configuration paths:
+```sh
+python3 -m edq.cli.config.list --config-file ./edq-config.json --config-file ~/.secrets/edq-config.json
+```
+
+#### CLI Configuration
+
+Configuration options are passed to the command line by the `--config` flag in this format `--config <key>=<value>`.
+The provided values overrides the values from configuration files.
+Configuration options are structured as key value pairs and keys cannot contain the "=" character.
+
+Below is an example of specifying a configuration option directly from the CLI:
+```sh
+python3 -m edq.cli.config.list --config user=alice --config pass=password123
+```
+
+#### CLI Config Options
+
+The table below lists all the default configuration CLI options available for CLI tools using this library.
+
+| CLI Option       | Description |
+| :--------------  | :---------- |
+|`--config-global` | Override the global config file location. |
+|`--config-file`   | Load configuration options from CLI specified files. |
+| `--config`       | For providing additional CLI configuration parameters to a CLI command. |
```

Comment:
Phrasing here does not match the other.
Try something like: "Provide additional ...".